src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.nocpm.props

```diff
@@ -9,7 +9,56 @@
 		<SynthesizeLinkMetadata>true</SynthesizeLinkMetadata>
 
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-	  	<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
+		<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
+
+		<DefaultLanguage>en</DefaultLanguage>
+
+		<!--#if (android)-->
```

Comment:
I think we can remove the conditions around these properties - if targets are added later, things will just work